src_py/__init__.py

```diff
@@ -178,59 +178,6 @@ def Cursor(*args):  # pylint: disable=unused-argument
     pixelcopy = MissingModule("pixelcopy", urgent=1)
 
 
-def warn_unwanted_files():
```

Comment:
I tracked this down in git, and it was added sometime between 1.8.1 and 1.9.0, and has been disabled since 1.9.1. So this all went down in 2008/2009, seems safe to remove in the present day.

Reply:
Safe to remove once the `del` of it is removed at the end of `__init__.py`, that is 😄

Reply:
Oh yeah, good catch

Reply:
And I thought I could get away with not testing a PR as "minor" as this. Good thing CI exists